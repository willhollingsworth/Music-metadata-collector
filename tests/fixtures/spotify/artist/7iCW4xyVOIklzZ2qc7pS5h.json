{
  "external_urls": {
    "spotify": "https://open.spotify.com/artist/7iCW4xyVOIklzZ2qc7pS5h"
  },
  "followers": {
    "href": null,
    "total": 23358
  },
  "genres": [
    "melodic house",
    "melodic techno",
    "progressive house"
  ],
  "href": "https://api.spotify.com/v1/artists/7iCW4xyVOIklzZ2qc7pS5h",
  "id": "7iCW4xyVOIklzZ2qc7pS5h",
  "images": [
    {
      "url": "https://i.scdn.co/image/ab6761610000e5eb230c754bdedd99ae1644300f",
      "height": 640,
      "width": 640
    },
    {
      "url": "https://i.scdn.co/image/ab67616100005174230c754bdedd99ae1644300f",
      "height": 320,
      "width": 320
    },
    {
      "url": "https://i.scdn.co/image/ab6761610000f178230c754bdedd99ae1644300f",
      "height": 160,
      "width": 160
    }
  ],
  "name": "Romain Garcia",
  "popularity": 51,
  "type": "artist",
  "uri": "spotify:artist:7iCW4xyVOIklzZ2qc7pS5h"
}