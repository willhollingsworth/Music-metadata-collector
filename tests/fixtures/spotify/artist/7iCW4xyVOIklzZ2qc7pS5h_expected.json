{
  "artist_name": "Romain Garcia",
  "artist_id": "7iCW4xyVOIklzZ2qc7pS5h",
  "followers": 23445,
  "popularity": 52,
  "albums": []
}